backend/pkg/api/data_access/vdb_slotviz.go

```diff
@@ -206,7 +205,7 @@ func (d *DataAccessService) GetValidatorDashboardSlotViz(ctx context.Context, da
 			}
 			attestationsRef := slotVizEpochs[epochIdx].Slots[slotIdx].Attestations
 
-			if uint64(slot) >= latestProposedSlot {
+			if uint64(slot) >= dutiesInfo.LatestSlot {
```

Comment:
are the changes in this file on purpose?

Reply:
nope :face_with_open_eyes_and_hand_over_mouth: